packages/loopring_v3/contracts/aux/transactions/TransactionReader.sol

```diff
@@ -15,119 +15,102 @@ import "../../core/impl/libtransactions/WithdrawTransaction.sol";
 /// @author Brecht Devos - <brecht@loopring.org>
 /// @dev Utility library to read transactions.
 library TransactionReader {
-    using BlockReader       for ExchangeData.Block;
+    using BlockReader       for bytes;
     using TransactionReader for ExchangeData.Block;
     using BytesUtil         for bytes;
 
     function readDeposit(
         ExchangeData.Block memory _block,
-        uint txIdx
+        uint txIdx,
```

Comment:
Seams these readXXX functions are not used anywhere.

Reply:
Yes, I guess we can remove them, although they may still be useful in other use cases (e.g. receipts).